### PR TITLE
Solvi/add basic about page

### PR DESCRIPTION
**Pull request by SolviQorda**
SUMMARY
Solves this user story - https://trello.com/c/OH90JhYY/14-as-a-user-i-want-to-be-able-to-get-involved-with-the-development-of-the-tracker-to-do-this-i-need-to-find-out-on-the-webapp-who

TASKS COMPLETED

- Adds about page with links to our Trello, Slack, GitHub and contact
- Adds route for About to `constants/routes`
- Adds About to main nav

TASKS STILL TO DO
Add a get involved form

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -48,6 +49,7 @@ const App = (props) => {
 						<Route exact path={routes.PROFILE} component={ProfilePage} />
 						<Route exact path={routes.USER_PAGE} component={UserPage} />
 						<Route exact path={routes.ORGANISATION_PAGE} component={OrganisationPage} />
+            <Route exact path={routes.ABOUT_PAGE} component={AboutPage} />
```
Comment (roborourke) [inline src/components/App.js:52]: Minor issue with tabs v spaces I think, not a big problem as it's kind of inconsistent across the board right now, will look into adding an `.editorconfig` / ESLint or something

File: src/components/Navigation/index.js [hunk 2]
```diff
@@ -7,73 +7,74 @@ import AccountDropdown from './AccountDropdown';
...
+//         </Menu.Item>
 //     </Menu>
 
 
-const NonAuthMenuItems = () => 
+const NonAuthMenuItems = () =>
     <Fragment>
         <Menu.Item exact as={NavLink} to={routes.HOME_PAGE}>Homepage</Menu.Item>
         <Menu.Item as={NavLink} to={routes.SIGN_IN}>Sign in</Menu.Item> 
+        <Menu.Item as={NavLink} to={routes.ABOUT_PAGE}>About</Menu.Item>
```
Comment (roborourke) [inline src/components/Navigation/index.js:48]: Would make sense for this to come after the home page link and before sign in I think.

**Review by roborourke — COMMENTED** [review]
Code all looks fine to me, added a couple of suggestions. I think before we merge this we need to get `master` up to date with codesandbox branch to make sure things don't get mixed up again, or alternatively create this pull request directly against the codesandbox branch.

File: src/components/App.js (hunk 1 above)
Comment (SolviQorda) [reply src/components/App.js:52]: Oh weird! That must have been my linter, which was also the source of the deleted whitespace. Will fix this.

File: src/components/Navigation/index.js (hunk 2 above)
Comment (SolviQorda) [reply src/components/Navigation/index.js:48]: sure :+1:

File: src/components/App.js (hunk 1 above)
Comment (SolviQorda) [reply src/components/App.js:52]: Well I just managed to make it worse. Is this going to break it, or shall I leave it for now

**Pull request merged**